.travis.yml

```diff
@@ -35,24 +35,26 @@ jobs:
 
     - stage: "Tests"
       name: "Tests"
+      install:
+        - yarn install --non-interactive
       script:
-        - npm run lint:js
-        - npm test
+        - yarn lint:hbs
```

Comment:
👋 @rwwagner90 . Once this is removed, I think the tests will pass!

Reply:
@snewcomer ah I forgot to add ember-cli-template-lint to package.json. Added that in, so tests should run now/

Reply:
Also could remove `start-app.js` from the test helpers.